ui/src/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentEditor.js

```diff
@@ -10,6 +11,31 @@ export class DocumentEditor extends Component {
     error: {},
   };
 
+  get documentRequest() {
+    const request = _get(this.props, 'location.state', null);
+    if (request) {
+      const data = {
+        documentRequestPid: request.pid,
+        metadata: {
+          // NOTE: Cheating so the authors don't break our document form
+          authors: [{ full_name: request.metadata.authors, type: 'PERSON' }],
+          title: request.metadata.title,
+        },
+      };
+      const journal_title = _get(request, 'metadata.journal_title', null);
```

Comment:
```suggestion
      const journalTitle = _get(request, 'metadata.journal_title', null);
```

Reply:
This one just follows JavaScript naming convention